releasenotes/notes/refresh-secrets-setting-1865ad684a8c27d0.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Config setting `secret_refresh_interval` will periodically refresh
+    secrets at the given interval.
```

Comment:
Same here: as the feature is not yet usable we should avoid documenting it for customers.